package.json

```diff
@@ -16,8 +16,6 @@
 		}
 	},
 	"scripts": {
-		"build": "node scripts/build.js",
-		"build:css": "node ./bin/css-builder/css-builder.js",
```

Comment:
@tn3rb we won't need this, right ?

Reply:
mmm no not now... I want to keep some of that code, but we will rework how it is implemented.
we'll most likely move it's implementation out of the build process and into some admin based services.